libs/coin-modules/coin-polkadot/src/network/sidecar.types.ts

```diff
@@ -13,23 +13,19 @@ interface IPallet {
   pallet: string;
   palletIndex: number;
 }
-interface IUnlocking {
+export interface IUnlocking {
```

Comment:
```suggestion
export interface Unlocking {
```
I think we moved away from this convention in the code base

Reply:
Yeah, it's a little bit too much "Java old style". But I prefer to not touch everything in this PR and stay focus on the main purpose (use node calls)